hazelcast/src/main/java/com/hazelcast/internal/probing/ProbeRenderer.java

```diff
@@ -0,0 +1,20 @@
+package com.hazelcast.internal.probing;
+
+import com.hazelcast.spi.annotation.Beta;
+import com.hazelcast.spi.annotation.PrivateApi;
+
+@Beta
```

Comment:
We don't need to put Beta on internals API's. We can break whatever needs to be broken.

Reply:
removed.